lib/virt_autotest/utils.pm

```diff
@@ -382,4 +386,86 @@ sub start_guests {
     wait_guest_online($_) foreach (keys %virt_autotest::common::guests);
 }
 
+#Add common ssh options to host ssh config file to be used for all ssh connections when host tries to ssh to another host/guest.
+sub setup_common_ssh_config {
+    my $ssh_config_file = shift;
+
+    $ssh_config_file //= '/root/.ssh/config';
+    assert_script_run("touch $ssh_config_file") if (script_run("test -f $ssh_config_file") ne 0);
+    if (script_output("cat $ssh_config_file | grep \"Host \\\*\"", proceed_on_failure => 1) eq '') {
```

Comment:
if (script_run "grep xxx $file" == 0) is the same, your command is ok as well(but 'cat file' seems to be redundant).

Reply:
I think this makes this no much difference. I just used another api script_output to compare whether output is empty which is also quite well.

Reply:
Although I think current call scirpt_output is ok as well, I already changed to use script_run.